.travis.yml

```diff
@@ -13,9 +12,7 @@ install:
   - pip install -U pip setuptools
   - pip install tox-travis
   - pip install codecov
-  - if [[ $TRAVIS_PYTHON_VERSION != 3.4 ]]; then pip install pre-commit; fi
```

Comment:
The whole line was removed but command `pip install pre-commit` needs to be there.

Reply:
oops - got overzealous removing 3.4